packages/suite/src/components/suite/modals/ReduxModal/ConfirmAddressModal.tsx

```diff
@@ -32,8 +27,6 @@ export const ConfirmAddressModal = ({ addressPath, value, ...props }: ConfirmAdd
     const device = useSelector(selectSelectedDevice);
     const account = useSelector(selectAccountIncludingChosenInTrading);
     const isTradingFlow = useSelector(selectTradingModalAccountKey);
```

Comment:
I think this can be also removed

Reply:
and those two params removed from `ConfirmValueModal` (when refactored accordingly)
```
          areStepsVisible={!isTradingFlow}
          isCopyButtonVisible={!isTradingFlow}
```